eurekaclient/servicelookup.json

```diff
@@ -0,0 +1 @@
+[{"uri": "/book", "app": "BOOK", "secure": false},{"uri":"/httpbin", "app": "BOOK", "secure": false}]
```

Comment:
This will help with testing. A sample json mapper (MG config won't be available). But if its not needed, I can remove it.